app/assets/stylesheets/_users.scss

```diff
@@ -0,0 +1,135 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+body {
+  font-family: candara;
+}
+
+$light-grey: lighten(#b2beb5, 10%);
+
+form {
+  border-right: 2px solid $light-grey;
+}
+
+.row.container {
+  box-shadow: none;
```

Comment:
Properties should be ordered bottom, box-shadow, left, margin, right, top, width